docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,27 +1,195 @@
-= 5.1 Release Notes
-:description: In this release, we introduced pod scheduling support and added Phone Home service to collect usage data to shape our roadmap.
+= 5.2 Release Notes
 
-{description}
+In this release, we introduced Backup & Restore to/from PVC and hostPath
+features.
 
 == New features
 
-* Pod scheduling support for Hazelcast Custom Resource (Node Selector, Node Affinity, Pod Affinity&Anti-affinity, and Tolerations)
-```
+* Backup data into
+https://kubernetes.io/docs/concepts/storage/persistent-volumes/[PVC]
+(via
+https://docs.hazelcast.com/hazelcast/latest/storage/persistence[Hazelcast
+Persistence])
++
+The persistence feature needs to be enabled at Hazelcast CR:
```

Comment:
We have docs for all these features. It's better to link to the doc than provide instructions in release notes. This applies to all instructions in this section.

Reply:
They are not instructions actually, give some idea/starting points about possible YAML usage to users. Providing some details to customer/users via release notes is a good idea, I think.

Reply:
In that case, we should consider the following:
- Add a link to the feature's doc so that users can learn more about how it works/how to use it.
- Avoid duplication by including the YAML snippets like we do here: https://github.com/hazelcast/hazelcast-platform-operator-docs/edit/main/docs/modules/ROOT/pages/backup-restore.adoc

Reply:
Good points, I will consider them during the preparation of 5.3 release notes :man_shrugging: